packages/parallax/src/index.tsx

```diff
@@ -274,22 +309,33 @@ export const Parallax = React.memo(
           ...rest.style,
         }}>
         {ready && (
-          <a.div
-            ref={contentRef}
-            style={{
-              overflow: 'hidden',
-              position: 'absolute',
-              [horizontal ? 'height' : 'width']: '100%',
-              [horizontal ? 'width' : 'height']: state.space * pages,
-              WebkitTransform: START_TRANSLATE,
-              msTransform: START_TRANSLATE,
-              transform: START_TRANSLATE_3D,
-              ...props.innerStyle,
-            }}>
+          <>
+            <a.div
+              ref={contentRef}
+              style={{
+                overflow: 'hidden',
+                position: 'absolute',
+                [horizontal ? 'height' : 'width']: '100%',
+                [horizontal ? 'width' : 'height']: state.space * pages,
+                WebkitTransform: START_TRANSLATE,
+                msTransform: START_TRANSLATE,
+                transform: START_TRANSLATE_3D,
+                ...props.innerStyle,
+              }}>
+              <ParentContext.Provider value={state}>
+                {React.Children.map(
```

Comment:
Couldn't this just be `rest.children.map(child => child.props.sticky && child)`?

Reply:
I thought so too, but I was getting a TypeScript error: `Property 'map' does not exist on type string` and so I thought `React.Children` was the way to go about it (also, it deals with the edge case of having only one `ParallaxLayer`, in which case mapping `children` would fail because it's not an array). I did manage to get it down to this just now (after destructing):
```
React.Children.map(children, (child: any) => child.props.sticky && child)
```
`children.map...` also works in this case, but will still fail to render anything if there is only one child. Though, to be fair, I can't see a lot of situations where there would only be one.

Reply:
Ah yes, one child, although you're right is a bit of a weird case.... Lets keep it with what you have ⭐  it was a tiny point anyway 👍🏼